CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   template, to the commit description. Use cases include DCO Sign Off and
   Gerrit Change Id.
 
+* `jj new` now uses the `commit_trailers` template.
```

Comment:
Might be a good idea to coalesce these "`jj [command]` now supports trailers" lines into one changelog entry as they're merged.